docs/production/install-existing-server.md

```diff
@@ -75,7 +75,13 @@ If you have an existing PostgreSQL database, note that Zulip will use
 the default `main` as its database name; make sure you're not using
 that.
 
-### Memcached, redis, and rabbitmq
+### rabbitmq
+
+rabbitmq will configure the `nodename` to be `rabbit@localhost`, in
+order to be resilient to hostname changes.  This will result in a
+change in data storage path for the node, if it had pre-existing data.
```

Comment:
I think the background for the messy code you deleted here is that we were originally migrating to this `rabbit@localhost` model from "whatever Ubuntu did by default", and were reluctant to break every existing system on upgrade through the change of RabbitMQ node.

This is probably a TODO for later, but we currently don't have a central place where we document the features of `zulip.conf`; if we had such a file, I'd add a link to it here.

Reply:
Ref #12490 it looks like.